app/templates/components/event-card.hbs

```diff
@@ -23,8 +23,18 @@
           {{moment-format event.startTime 'ddd, MMM DD HH:mm A'}}
         </span>
       </div>
-      <div class="description">
```

Comment:
Sorry about that class location! I unintentionally forgot to remove it. It has nothing to do. I was just experimenting with the things hence forgot to remove. Yes, I'll make changes for that and the description class to be available for entire condition.